packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx

```diff
@@ -99,8 +105,10 @@ const IdentifierInput: React.FC<IdentifierInputProps> = ({ patientIdentifier, fi
         />
       ) : (
         <div className={styles.textID}>
-          <p className={styles.label}>{identifierName}</p>
-          <p data-testid="identifier-label" className={styles.bodyShort02}>
+          <p data-testid="identifier-label" className={styles.label}>
+            {required ? identifierName : `${identifierName} (${t('optional', 'optional')})`}
```

Comment:
Different languages have different requirements around word order, so it would be better to make this whole thing translatable. See [the i18next docs](https://www.i18next.com/translation-function/interpolation).